pallets/liquidity-pools/src/tests/inbound.rs

```diff
@@ -227,6 +235,47 @@ mod handle_tranche_tokens_transfer {
 			})
 		}
 
+		#[test]
+		fn inbound_with_frozen_investor_permissions() {
+			System::externalities().execute_with(|| {
+				config_mocks(CENTRIFUGE_DOMAIN_ADDRESS);
+				Permissions::mock_has(|_, _, _| true);
+
+				assert_noop!(
+					LiquidityPools::handle(
+						EVM_DOMAIN_ADDRESS,
+						Message::TransferTrancheTokens {
+							pool_id: POOL_ID,
+							tranche_id: TRANCHE_ID,
+							domain: CENTRIFUGE_DOMAIN_ADDRESS.domain().into(),
+							receiver: ALICE.into(),
+							amount: AMOUNT,
+						}
+					),
+					Error::<Runtime>::InvestorDomainAddressFrozen,
+				);
+			})
+		}
+
+		#[test]
+		fn outbound_with_frozen_investor_permissions() {
```

Comment:
Should it be in the `tests.rs` which check the extrinsic calls?

Reply:
[a5f2e6e](https://github.com/centrifuge/centrifuge-chain/pull/1947/commits/a5f2e6e272c2b8400190994a4529f97ca012ad46)